platform/windows/os_windows.cpp

```diff
@@ -1568,10 +1569,19 @@ void OS_Windows::set_window_position(const Point2& p_position){
 	MoveWindow(hWnd,p_position.x,p_position.y,r.right-r.left,r.bottom-r.top,TRUE);
 
 }
+
+void OS_Windows::show_window() {
+	if (!is_no_window_mode_enabled()) {
+		ShowWindow(hWnd,SW_SHOW);						// Show The Window
+		SetForegroundWindow(hWnd);						// Slightly Higher Priority
+		SetFocus(hWnd);									// Sets Keyboard Focus To
+	}
+}
+
 Size2 OS_Windows::get_window_size() const{
 
 	RECT r;
-	GetClientRect(hWnd,&r);
+	GetWindowRect(hWnd,&r);
```

Comment:
I as far as I understand, this change is needed for `OS::set_window_centered` to work correctly and actually center the window... For that it needs to include the titlebar, borders, etc.., that's true.
But I believe this is the wrong way to solve that problem. It will introduce #4851 on the windows platform.
I think what you should do is leave `get_window_size()` as it is and instead override `set_window_centered()` in `OS_Windows` (which uses the correct sizing).